app/services/irs_api_service.rb

```diff
@@ -15,11 +15,13 @@ def self.df_return_sample
   def self.import_federal_data(authorization_code, _state_code)
     unless Rails.env.production?
       xml_return_sample_service = StateFile::XmlReturnSampleService.new
+      json_return_sample_service = StateFile::JsonReturnSampleService.new
       if xml_return_sample_service.include?(authorization_code)
         return {
           'xml' => xml_return_sample_service.read(authorization_code),
           'submissionId' => xml_return_sample_service.lookup_submission_id(authorization_code),
-          'status' => "accepted"
+          'status' => "accepted",
+          'directFileData' => json_return_sample_service.read(authorization_code)
```

Comment:
we can probably just do 
`StateFile::JsonReturnSampleService.new.read(authorization_code)` instead of defining it, since its only used once